### PR TITLE
Changelog 4.0.0/team thread sort

### PR DESCRIPTION
**Pull request by ajpowelsnl**
This PR simply adds #5317 as an item to the Kokkos-4.0.0 CHANGELOG.  This item will be featured in the upcoming Demo.

### CODE REVIEW

**Comment by masterleinad** [comment]
Surely, you didn't mean to add 446 commits. 🙂

**Comment by ajpowelsnl** [comment]
> Surely, you didn't mean to add 446 commits. 🙂

Correct.

**Comment by dalg24** [comment]
PR must target the `develop` branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - `parallel_scan` with `View` as result type. [\#5146](https://github.com/kokkos/kokkos/pull/5146)
 - Introduced `SharedSpace`, an alias for a `MemorySpace` that is accessible by every `ExecutionSpace`. The memory is moved and then accessed locally. [\#5289](https://github.com/kokkos/kokkos/pull/5289)
 - Introduced `SharedHostPinnedSpace`, an alias for a `MemorySpace` that is accessible by every `ExecutionSpace`. The memory is pinned to the host and accessed via zero-copy access. [\#5405](https://github.com/kokkos/kokkos/pull/5405)
+- Team- and Thread-level sort, sort_by_key.  The interfaces to sort_[team/thread] and sort_by_key[team/thread] are designed to be similar to Thrust, except they take Kokkos::Views and not iterators. [\#5317](https://github.com/kokkos/kokkos/pull/5317)
```
Comment (dalg24) [inline CHANGELOG.md:11]:
```suggestion
- Add team- and thread-level `sort` and `sort_by_key` algorithms. [\#5317](https://github.com/kokkos/kokkos/pull/5317)
```

**Review by dalg24 — COMMENTED** [review]
Please use `[ci skip]` in your commit message

**Review by dalg24 — CHANGES_REQUESTED** [review]
Still targeting the wrong branch

**Comment by crtrott** [comment]
You can't change master anymore, this probably should be a fix in the patch release 4.0.01

**Comment by masterleinad** [comment]
@ajpowelsnl ping

**Comment by ajpowelsnl** [comment]
> You can't change master anymore, this probably should be a fix in the patch release 4.0.01

I'll add #6007 to the 4.0.01 CHANGELOG

**Comment by dalg24** [comment]
@ajpowelsnl Your PR is against the master branch instead of develop.  Please fix it.

**Comment by masterleinad** [comment]
@ajpowelsnl ping

**Comment by ajpowelsnl** [comment]
Close b/c needs to be on `develop`

**Comment by dalg24** [comment]
Would need to target 4.0.01 and is probably too late
cc @ndellingwood

**Pull request closed without merging**